specs/MultiProfile.md

```diff
@@ -217,6 +232,80 @@ HRESULT AppWindow::OnCreateCoreWebView2ControllerCompleted(
 }
 ```
 
+### Manage password-autosave and general-autofill settings in profile
+
+```cpp
+HRESULT AppWindow::ManagePasswordAutosaveInProfile(ICoreWebView2Controller* controller)
+{
+    // ...
+
+    // Get the profile object.
+    wil::com_ptr<ICoreWebView2> coreWebView2;
+    CHECK_FAILURE(m_controller->get_CoreWebView2(&coreWebView2));
```

Comment:
Is this really m_controller and not the controller param?

Reply:
It should be `controller`, updated the code.